apps/client/app/purchase-orders/page.tsx

```diff
@@ -1,8 +1,92 @@
+import SelectList from "../components/selectList";
+import { getFormattedDate, getParentItems, getPurchaseOrders } from "../utils";
+
+const sortByArr = [
+  {
+      key: "expected_delivery_date",
+      value: "Delivery Date",
+  },
+  {
+      key: "vendor_name",
+      value: "Vendor Name",
+  },
+  {
+      key: "order_date",
+      value: "Order Date",
+  },
+];
+
+export default async function Index({ searchParams }: any) {
+  const { sort } = searchParams;
+  let purchaseOrders = await getPurchaseOrders(sort);
+  const parentItems = await getParentItems();
+  
+
+  
+  let items: Item[] = [];
+
+  // Hacky, but but needed a quick way to get all items from the db.
+  parentItems.forEach(parentItem => (
+    items.push(...parentItem.items)
+  ))
+
+
 
-export default async function Index() {
   return (
     <>
       <h1 className="text-2xl">Purchase Orders</h1>
+      <div className="">
+        <SelectList list={sortByArr} selected={sort} />
+      </div>
+      <table className="border-collapse table-auto w-full text-sm">
+        <thead>
```

Comment:
Reusing the same table and styles as the parent items page. With a growing project I would like to create a table component that takes in headers and a list of data then creates the table as we like. This would allow us to keep the same styles and format throughout the app.